infrastrucutre/vega-app-helm-chart/templates/ingress.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     alb.ingress.kubernetes.io/target-type: {{ .Values.ingress.type }}
     alb.ingress.kubernetes.io/backend-protocol: {{ .Values.ingress.backend_protocol }}
     alb.ingress.kubernetes.io/load-balancer-attributes: "idle_timeout.timeout_seconds=60"
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
 spec:
```

Comment:
You are missing the TLS part in order for ingress to be able to find certificate automatically.

Check the docs please:
https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/guide/ingress/cert_discovery/